src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThread.scss

```diff
@@ -1,7 +1,6 @@
 @import '../../../../styles/variables';
 
 .AnnotationThread {
-    max-width: 340px;
-    max-height: 600px;
+    min-width: $bdl-grid-unit * 85;
```

Comment:
We can probably revert this unless `85` is a meaningful value for our grid.

Reply:
I don't think it's a meaningful value. I thought that the `$bdl-grid-unit` is a "base" grid size value and the width of elements should be connected to it. I've changed to a static value.